support-frontend/assets/pages/weekly-subscription-landing/components/weeklyForm.jsx

```diff
@@ -23,28 +20,8 @@ import { getOrigin } from 'helpers/url';
 
 // ---- Plans ----- //
 
-const getCheckoutUrl = ({ billingPeriod, state }: {billingPeriod: WeeklyBillingPeriod, state: CommonState}): string => {
-  const optimizeExperimentId = 'bCiv10arQ7OYNMyIyC89MQ';
-  const {
-    internationalisation: { countryGroupId }, referrerAcquisitionData, abParticipations, optimizeExperiments,
-  } = state;
-
-  const useVariant = state.optimizeExperiments.find(exp => exp.id === optimizeExperimentId && exp.variant === '1');
-
-  if (useVariant) {
-    return `${getOrigin()}/subscribe/weekly/checkout?period=${billingPeriod.toString()}`;
-  }
-
-  return getWeeklyCheckout(
-    referrerAcquisitionData,
-    billingPeriod,
-    countryGroupId,
-    abParticipations,
-    optimizeExperiments,
-    (billingPeriod === 'Annual' ? getPromoCode('GuardianWeekly', countryGroupId, '10ANNUAL') : null),
-  );
-
-};
+const getCheckoutUrl = (billingPeriod: WeeklyBillingPeriod): string =>
```

Comment:
do we also need to pass through existing parameters?  Related to this current card where we haven't done that for print https://trello.com/c/MaLGg6Xq/2443-utm-parameters-not-passed-through-to-print-checkout-breaking-channel-reporting

Reply:
I've put a note on that card to say that whatever the solution is it needs to be applied to GW checkout as well. I'd rather do that than make a change here without investigating fully.

Reply:
Sounds good.